dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md

```diff
@@ -2,31 +2,29 @@
 uid: Overview_DCP
 ---
 
-# DataMiner Cloud Platform
+# dataminer.services
 
-The DataMiner Cloud Services represent a genuine paradigm shift, offering innovative and game-changing solutions, which support the transition into the emerging ICT media and broadband landscape and empower digital transformation programs.
-
-The name **DataMiner Cloud Services** refers to the combined overall cloud ecosystem, which consists of different components:
+With dataminer.services, you gain access to innovative and game-changing solutions that support the transition into the emerging ICT media and broadband landscape and empower digital transformation programs. These services include the following components:
 
 - [Cloud-connected DataMiner Systems](#cloud-connected-dataminer-systems)
```

Comment:
Remote access - it refers to this topic https://docs.dataminer.services/user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.html